mycity/mycity/intents/feedback_intent.py

```diff
@@ -99,3 +99,26 @@ def build_slack_message(feedback_type, feedback_text):
     )
     emoji = ':bug:' if feedback_type == 'bug' else ':bulb:'
     return emoji + '\n>' + feedback_text
+
+
+def build_slack_traceback(error, trace):
+    """
+    Configures the message we will post to slack
+
+    :param error:
+    :return:
```

Comment:
Can you add the docstring here?

Reply:
How do you mean? What's missing of the docstring?